scripts/run_test_plan.sh

```diff
@@ -422,8 +430,35 @@ kubectl top pods -n flotta --use-protocol-buffers
 } >> $test_dir/summary.txt
 }
 
+setup()
```

Comment:
this is repeated in `setup.sh` script as well. Let's invoke that script instead of defining the needed configuration in both places.